tests/keras_tests/feature_networks_tests/feature_networks/mixed_percision_test.py

```diff
@@ -199,3 +199,41 @@ def get_quantization_config(self):
                                     activation_channel_equalization=False)
 
         return MixedPrecisionQuantizationConfig(qc)
+
```

Comment:
If I understand correctly, this file contains only tests of weights MP. If this is the situation, let's rename the file to indicate it.

Reply:
No problem, but beware that the new test file 'mixed_precision_activation_test.py' has tests for both (I don't disable the weights mp there on purpose, so it might be a bit confusing)

Reply:
Change to "weights_mixed_precision_tests"